spec/models/comfy/cms/page_spec.rb

```diff
@@ -251,6 +251,22 @@
     end
   end
 
+  describe '#scheduled_today scope' do
```

Comment:
Great tests but the structure of the test can be re-organised to communicate the true intention of the test. 

```
describe '#scheduled_today' do
  context 'pages scheduled in the past' do
    it 'includes only pages scheduled for today' do
       ...
    end
  end
end
```